### ISSUE TITLE
Zoltan2: Adapter Build Failure

### ISSUE THREAD

**Issue by cwschilly** [NexGenAnalytics/Trilinos#19]
In `Zoltan2_TpetraRowMatrixAdapter.hpp`, `getRowIDsDeviceView` does not compile

**Comment by stmcgovern** [comment]
This issue is superseded by #20 for issue #13 . It can be closed once that PR is merged.

**Comment by cwschilly** [comment]
#13 has been closed
